storybook/src/Util.elm

```diff
@@ -0,0 +1,9 @@
+module Util exposing (..)
+
+import Html exposing (Attribute, Html)
+import Html.Attributes
+
+
+col : List (Attribute msg) -> List (Html msg) -> Html msg
+col attrs children =
+    Html.div (Html.Attributes.class "col" :: attrs) children
```

Comment:
I think this is useful enough to be added to either a new `UI.Layout` module or just as a function on the `UI` module.